.github/workflows/long-running-azure.yaml

```diff
@@ -82,9 +84,9 @@ env:
   VALID_RADIUS_BUILD_WINDOW: 86400
 
   # The AKS cluster name
-  AKS_CLUSTER_NAME: "radiuse2e00-aks"
+  AKS_CLUSTER_NAME: "radlrtest00-aks"
```

Comment:
> Would these names be better as Actions Variables instead of hard-coded in the workflow file? Would allow us to quickly update the values without needing to check in a change.

Sure. Please add new variable for cluster name. I can update it.

Reply:
Since these would probably be best as repo-scoped variables, you should be able to edit them here: https://github.com/radius-project/radius/settings/variables/actions. Lemme know if you hit any access issues.

Reply:
> Since these would probably be best as repo-scoped variables, you should be able to edit them here: https://github.com/radius-project/radius/settings/variables/actions. Lemme know if you hit any access issues.

I do not have the permission.

Reply:
I merged it now. I will create new pr to use new var.